frontend/packages/pipelines-plugin/src/utils/__tests__/pipeline-utils.spec.ts

```diff
@@ -144,6 +144,23 @@ describe('pipeline-utils ', () => {
     });
   });
 
+  it('expect annotation to have correct prefix when there are existing annotations', () => {
+    const existingAnnotations = {
+      'tekton.dev/git-0': 'gitlab.com',
+    };
+    const annotations = getSecretAnnotations(
+      {
+        key: SecretAnnotationId.Git,
+        value: 'github.com',
+      },
+      existingAnnotations,
+    );
+    expect(annotations).toEqual({
+      'tekton.dev/git-0': 'gitlab.com',
+      'tekton.dev/git-1': 'github.com',
+    });
+  });
```

Comment:
Could you add one test case for avoiding duplicates and also one for incorrect `annotation.key`  
eg: 
```
getSecretAnnotations(
   {
     key: 'not-a-real-key', //  now it is returning {"null" : 'github.com'} but it should be {}
     value: 'github.com',
     },
    existingAnnotations,
 );
 ```